packages/flutter/lib/src/material/tabs.dart

```diff
@@ -584,10 +584,131 @@ class _TabBarScrollController extends ScrollController {
 /// Uses values from [TabBarTheme] if it is set in the current context.
 ///
 /// To see a sample implementation, visit the [TabController] documentation.
+/// 
+/// {@tool dartpad --template=stateless_widget_material}
+///
+/// This sample shows the implementation of [TabBar] with a [DefaultTabController]
+/// and a [TabBar] consisting of three [Tab] widgets with an [Icon] as a child.
+///
+///```dart
+/// Widget build(BuildContext context) {
+///    return DefaultTabController(
+///      initialIndex: 1,
+///      length: 3,
+///      child: Scaffold(
+///        appBar: AppBar(
+///          title: Text("TabBar Widget"),
+///          bottom: TabBar(
+///            tabs: [
+///              Tab(
+///                icon: Icon(
+///                  Icons.cloud_outlined,
+///                ),
+///              ),
+///              Tab(
+///                icon: Icon(
+///                  Icons.beach_access_sharp,
+///                ),
+///              ),
+///              Tab(
+///                icon: Icon(
+///                  Icons.brightness_5_sharp,
+///                ),
+///              ),
+///            ],
+///          ),
+///        ),
+///        body: TabBarView(
+///          children: [
+///            Center(
+///              child: Text(
+///                "It's cloudy here",
+///              ),
+///            ),
+///            Center(
+///              child: Text(
+///                "It's rainy here",
+///              ),
+///            ),
+///            Center(
+///              child: Text(
+///                "It's sunny here",
+///              ),
+///            ),
+///          ],
+///        ),
+///      ),
+///    );
+///  }
+///```
+///
+/// [TabBar] can also be implmented by using a [TabController] which provides more options,
+/// instead of a [DefaultTabController], below is a sample for this.
```

Comment:
note that when this gets generated, the two sample codes are merged into one file. I think you want to have two separate @tool blocks.

Reply:
True story, @sameerkash this is what is causing the checks to fail.

Reply:
@Piinks Thanks for reviewing this, I had fixed this but looks like I messed up while rebasing and didn't see it was overwritten.
I've fixed it now, But still, my checks seem to be failing. Could there be any reason?